shared/src/main/scala/mouse/option.scala

```diff
@@ -25,4 +28,6 @@ final class OptionOps[A](private val oa: Option[A]) extends AnyVal {
    * `Serializable with Product with Either[A, B]`
    */
   @inline def left[B](b: => B): Either[A, B] = oa.toLeft(b)
+
+  @inline def toOptionT[F[_]: Functor]: OptionT[F, A] = OptionT.liftF(oa)
```

Comment:
`cats` contains the same method in the `Option` syntax https://github.com/typelevel/cats/blob/main/core/src/main/scala/cats/syntax/option.scala#L368